bin/start

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Ensure Foreman is installed
+foreman --help &> /dev/null || gem install foreman
```

Comment:
`gem install` assumes ruby is present on the OS. This should also be documented in the app's readme so devs know to install it ahead of time.

Reply:
Fair point. I enjoy using `foreman` because it just works with a Procfile and lets you spin up both the frontend and server in the same process.